src/style.css

```diff
@@ -0,0 +1,80 @@
+body {
+  margin: 0 8px;
+}
+
+.common_input {
+  font-family: "Avenir-book", sans-serif;
+  font-weight: 300;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.common_input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.common_form {
+  width: 100%;
+}
```

Comment:
width is redundant for form
<img width="1438" alt="Screenshot 2022-03-21 at 16 49 36" src="https://user-images.githubusercontent.com/37452763/159286940-4011d88d-4530-4476-9a63-c5a32dfe4174.png">